sources/packages/web/src/components/students/applicationTracker/Cancelled.vue

```diff
@@ -1,13 +1,18 @@
 <template>
   <application-status-tracker-banner
-    label="You've cancelled your application"
+    label="Your application was cancelled"
     icon="fa:fas fa-exclamation-circle"
     icon-color="danger"
     background-color="error-bg"
     ><template #content
-      >You cancelled your application on
+      >Your application was cancelled on
       {{ dateOnlyLongString(cancelledDate) }}. However you can still view your
-      application by clicking on the “Application actions” button.
+      application by clicking on the "Application actions" button. If you did
+      not cancel your application, it means that your application has been
```

Comment:
@hellolynn-tbtb I am not sure how accurate this is or how close to production need it will be.
```
it means that your application has been canceled due to changes in the study period made by your institution.
```
In my opinion, it could be more generic and not point to the institution as the only possible point of responsibility for the change.

Reply:
Thanks @andrewsignori-aot. Yeah we may need it to be more generic if it contains other scenarios. But I am okay to leave it as is since we still have the "content review" track with business.